src/Enzo/enzo_EnzoMethodMHDVlct.hpp

```diff
@@ -268,6 +270,28 @@ class EnzoMethodMHDVlct : public Method {
    EnzoBfieldMethod *bfield_method, int stale_depth,
    const str_vec_t& passive_list) const noexcept;
 
+  /// Saves the fluxes (for a given dimension, `dim`), computed at the faces
+  /// between the active and the ghost zones to `block->data()->flux_data()`
+  /// for later use in flux corrections.
+  ///
+  /// This function technically saves the value of the flux multiplied by dt
+  /// and divided by the cell_width along dimension `dim`.
+  ///
+  /// @note
+  /// This does not currently support flux-correction equivalents for magnetic
+  /// fields.
```

Comment:
I'm not sure that the corrections on the EMF will use this function. I believe the edge EMFs need to be made consistent before taking the curl to update the magnetic fields, or inside of `compute`.  I'm not sure that correcting the magnetic fields after this curl would be reliable.

Reply:
Got it. I got the sense that what you're describing is how its handled in Athena++ from their AMR paper.

With that said, I'm inclined to keep the comment in-place until you implement the EMF flux corrections, just to have a note in the code to indicate that flux corrections are not applied to magnetic fields. But let me know if you want me to change what the comment actually says.

Once you have settled on your algorithm, it might be useful talking about how you're going to implement it. I suspect that this approach will probably involve splitting up the body of the ``for`` loop in ``EnzoMethodMHDVlct::compute`` into a sequence of 2 or 3 entry methods that get executed before/after the information needed for corrections is communicated between blocks. If you prefer, of course, you could make all the changes independently.

Reply:
You can leave the comment as is, it communicates that it doesn't work for magnetic fields but that flux correction for magnetic fields will be addressed in the future.

Yes, I'm planning on following very closely the EMF correction step in Athena++. For that to work, you do need to make the EMFs consistent between when you compute them and when you take flux divergences. I'm not that familiar with `charm++` nor making changes to Enzo-E, so the suggestion to split entry methods is very helpful.